rs/rosetta-api/icp/tests/system_tests/test_cases/neuron_management.rs

```diff
@@ -388,7 +388,7 @@ fn test_start_and_stop_neuron_dissolve() {
         let neuron = list_neurons(&agent).await.full_neurons[0].to_owned();
         assert!(
             matches!(
-                neuron.dissolve_state.unwrap(),
+                neuron.dissolve_state.clone().unwrap(),
```

Comment:
This is a small (but nonzero) difference between the two types: one is Copy, but the other is not. This illustrates how it is often easy to adjust for such differences.